YouVsVirus/Assets/Scripts/Components/EndLevelControllerBase.cs

```diff
@@ -42,7 +46,7 @@ public abstract class EndLevelControllerBase : MonoBehaviour
     /// This variable stores whether the level has
     /// already finished.
     /// </summary>
-    protected bool levelHasFinished = false;
+    public bool levelHasFinished = false;
```

Comment:
Maybe also make this protected again and add a setter/getter function?

Since if the level is finished we dont want anybody to set it to unfinished from the outside.

Reply:
Done with 54a33f274d6085b1f20d557ffda1ed1ef4fdcc26

Reply:
here it is 0b1565d536fbf4aaeee875e6ef1dfad77172deeb

Reply:
Done I think